packages/mobile/src/identity/verification.ts

```diff
@@ -164,7 +164,7 @@ export function* doVerificationFlow() {
     const autoRetrievalTask: Task = yield fork(startAutoSmsRetrieval)
 
     // This needs to go before revealing the attesttions because that depends on the public data key being set.
-    yield call(setAccount, attestationsContract, account, dataKey)
+    yield fork(setAccount, attestationsContract, account, dataKey)
```

Comment:
Nit: If we'd like to parallelize this can you restore to how it was before @trianglesphere made it serial?
https://github.com/celo-org/celo-monorepo-old/commit/b5323674642bbdf2e24df25e864b15609faf4037#diff-f08ccd136c52768094e71b0392941266

@trianglesphere can you remind us why that was necessary?

Reply:
I was seeing some odd errors when not setting the data encryption key. I believe that I tracked it down to something in `verify and reveal`. Skimming through what is there right now, I only see a reference to the data encryption key of the issuer, which should be set not by the client.

If it's working in parallel, the more I look at it, the more I think I'm fine with it if testing doesn't show any issues.